contracts/l1-l2-messaging/solidity/src/L1L2.sol

```diff
@@ -26,36 +26,33 @@ contract L1L2Example {
         starknetCore = starknetCore_;
     }
 
-    function get_balance(uint256 user)
```

Comment:
Unable to get the balance of a specified user because it uses "uint" data type instead of "address".